From 540836801f4ab5e8be9703d2c1350e988b2ccc1f Mon Sep 17 00:00:00 2001
From: "Jason M. Bills" <jason.m.bills@linux.intel.com>
Date: Thu, 11 Jun 2020 13:00:15 -0700
Subject: [PATCH] Extend VR Watchdog timeout

The VR watchdog reset is causing issues on platforms such as
Cooper City that take longer to assert CPU Power Good.  This
extends the timeout to 12s to hold off the reset for Cooper
City.

Change-Id: I9658b4ead6d9bf8eaa30e4aeb9f1d56c2f2187d3
Signed-off-by: Jason M. Bills <jason.m.bills@linux.intel.com>
---
 power-control-x86/config/power-config-host0.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/power-control-x86/config/power-config-host0.json b/power-control-x86/config/power-config-host0.json
index ca9a86a..f579c2b 100644
--- a/power-control-x86/config/power-config-host0.json
+++ b/power-control-x86/config/power-config-host0.json
@@ -18,7 +18,7 @@
       "ForceOffPulseMs": 15000,
       "ResetPulseMs": 500,
       "PowerCycleMs": 5000,
-      "SioPowerGoodWatchdogMs": 1000,
+      "SioPowerGoodWatchdogMs": 12000,
       "PsPowerOKWatchdogMs": 8000,
       "GracefulPowerOffS": 300,
       "WarmResetCheckMs": 500,
-- 
2.17.1

